### PR TITLE
Fix typo

### PR DESCRIPTION
**Pull request by zw963**
#725

### CODE REVIEW

File: src/elisp/treemacs-header-line.el [hunk 1]
```diff
@@ -69,10 +69,10 @@
                nil
                `(["Edit Workspaces"       treemacs-edit-workspaces]
                  ["Create Workspace"      treemacs-create-workspace]
-                 ["Remove Worspace"       treemacs-remove-workspace]
```
Comment (Alexander-Miller) [inline src/elisp/treemacs-header-line.el:72]: The values aligning as before is more important than a minimal diff.

File: src/elisp/treemacs-header-line.el [hunk 2]
```diff
@@ -69,10 +69,10 @@
...
+                 ["Remove Workspace"       treemacs-remove-workspace]
```
Comment (Alexander-Miller) [inline src/elisp/treemacs-header-line.el:72]: The commit message should be prefixed with `[Misc]`.

**Review by Alexander-Miller — CHANGES_REQUESTED** [review]

File: src/elisp/treemacs-header-line.el (hunk 1 above)
Comment (zw963) [reply src/elisp/treemacs-header-line.el:72]: Fixed.

**Comment by Alexander-Miller** [comment]
:+1:

File: src/elisp/treemacs-mouse-interface.el [hunk 3]
```diff
@@ -242,12 +242,12 @@ and ignore any prefix argument."
...
                 ("Workspaces"
-                 ["Edit Workspaces"       treemacs-edit-workspaces]
-                 ["Create Workspace"      treemacs-create-workspace]
-                 ["Remove Worspace"       treemacs-remove-workspace]
-                 ["Rename Workspace"      treemacs-rename-workspace]
-                 ["Switch Worspaces"      treemacs-switch-workspace]
-                 ["Set Fallback Worspace" treemacs-set-fallback-workspace])
+                 ["Edit Workspaces"        treemacs-edit-workspaces]
+                 ["Create Workspace"       treemacs-create-workspace]
+                 ["Remove Worskpace"       treemacs-remove-workspace]
```
Comment (chessman) [inline src/elisp/treemacs-mouse-interface.el:247]: that's another typo

Comment (Alexander-Miller) [reply src/elisp/treemacs-mouse-interface.el:247]: Doh. 

Fixing it now.

**Pull request merged**